### DISCUSSION TITLE
Limiting emails sent per domain

### DISCUSSION THREAD

**Discussion by kmagumise** [postalserver/postal#1500] [Feature suggestions]
Is there a way of limiting how many emails a domain can send. i.e. a domain can have 200000 email credits that have to be topped up when they reach that limit.

**Comment by mateuszulewicz** [comment]
I can write you such an add-on and other mods for Postal. And I have a lot of them and mega developed tool already :) of course at your disposal and your actions.

**Reply by estoreecom** [reply]
I need Your Service Ping me on skype live:mygreektech

**Reply by jshapiro2** [reply]
@yolmac Can you also ping me on skype @shap906 ?
